recipes/kaks_calculator/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: kaks_calculator
+  version: '2.0'
+
+source:
+  sha256: e2df719a2fecc549d8ddc4e6d8f5cfa4b248282dca319c1928eaf886d68ec3c5
+  url: https://sourceforge.net/projects/kakscalculator2/files/KaKs_Calculator2.0.tar.gz
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('kaks_calculator', max_pin="x") }}
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+
+test:
+  commands:
+    - KaKs_Calculator -h 2>&1 | grep "KaKs_Calculator" > /dev/null
+
+about:
+  home: 'http://sourceforge.net/projects/kakscalculator2/'
+  license: GPL-3.0-or-later
```

Comment:
where did you found the License information? Is GPL or later correct?
We need to add the license_file as well